.github/workflows/update.yaml

```diff
@@ -0,0 +1,13 @@
+name: Update gh-pages branch on alire.ada.dev
+on:
+  schedule:
+    - cron:  '0 0 * * *'
+jobs:
+  build:
+    runs-on: ubuntu-latest
```

Comment:
We are doing CI with ubuntu-18.04 (and debian stable) so that may be using an untested gnat-gcc?